Cargo.toml

```diff
@@ -37,7 +37,7 @@ clap                        = "4.5.41"
 config                      = "0.14.0"
 dashmap                     = "6.1.0"
 dotenvy                     = "0.15.7"
-fastcrypto                  = "0.1.8"
+fastcrypto                  = { git = "https://github.com/MystenLabs/fastcrypto", rev = "69d496c71fb37e3d22fe85e5bbfd4256d61422b9", package = "fastcrypto" }
```

Comment:
why ?

Reply:
because sui doesn't use versioned verion of fastcrypto they use the git version with specific revision. Using the crate 0.1.8 will result in incompatibility issues of traits. When they switch, we will switch.
their cargo https://github.com/MystenLabs/sui/blob/testnet-v1.52.2/Cargo.toml#L628